pydicom/tests/test_valuerep.py

```diff
@@ -675,6 +670,13 @@ def test_hash(self):
 
 
 class TestDSdecimal:
+    @pytest.fixture
+    def allow_ds_float(self):
+        old_value = config.allow_DS_float
```

Comment:
Had to add this to several tests as they only worked accidentally due to the execution order.